src/art/skypilot/backend.py

```diff
@@ -88,6 +98,11 @@ async def initialize_cluster(
             art_server_task.set_resources(cast(sky.Resources, resources))
             art_server_task.update_envs(self._envs)
 
+            # If a local path was provided for art_version, ensure it is mounted so the latest
+            # code is synced to the remote cluster every time we (re)launch the art_server task.
+            if art_version is not None and os.path.exists(art_version):
+                art_server_task.set_file_mounts({"~/sky_workdir": art_version})
```

Comment:
Hmm, not sure we want to do this by default. It's conceivable that I might be editing code on the cluster itself, and I don't want that getting overwritten.

Maybe make this a parameter, and default to False?

Reply:
As discussed we'll experiment with keeping this on by default for now and can turn it off if it ends up being annoying for @arcticfly's workflow.